package.json

```
@@ -15,6 +15,7 @@
     "test": "npm run test-unit",
     "posttest": "npm run lint",
     "test-api": "mocha \"**/*.api.js\"",
+    "test-addons": "mocha \"**/*.test.js\"",
```

I think you left this in accidentally

---

It is accidentally left here.
 but I found there's no such a command to run unit test for addons folder. https://github.com/xtermjs/xterm.js/blob/master/bin/test.js will only run test under `out` folder. 
So how about this
```suggestion
    "test-addons": "mocha \"addons/**/*.test.js\"",
```

---

If addons ever add unit tests (vs api tests) we can add the glob to the array in https://github.com/xtermjs/xterm.js/blob/master/bin/test.js, that way they would get run as part of `yarn test`

---

Agree